synapse/replication/slave/storage/room.py

```diff
@@ -12,26 +12,12 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from synapse.replication.tcp.streams import PublicRoomsStream
-from synapse.storage.database import DatabasePool
 from synapse.storage.databases.main.room import RoomWorkerStore
 
 from ._base import BaseSlavedStore
-from ._slaved_id_tracker import SlavedIdTracker
 
 
 class RoomStore(RoomWorkerStore, BaseSlavedStore):
```

Comment:
I wonder if we should just directly use `RoomWorkerStore` in `generic_worker`. We seem to do a mixture though so I wouldn't block on that.

Reply:
We do import the `*WorkerStore` variant in `GenericWorkerSlavedStore` in some cases, so doing so here probably isn't too contentious.

Where is this mixture you mentioned?

Reply:
I see, OK. I've removed the class for now as it's probably best to not keep it around unless needed.

Tests pass, though it probably warrants a double-check to make sure I did the right thing :)